components/camera/client.go

```diff
@@ -118,6 +118,8 @@ func getExtra(ctx context.Context) (*structpb.Struct, error) {
 	return ext, nil
 }
 
+// RSDK-8663: This method signature is depended on by the `camera.serviceServer` optimization that
+// avoids using an image stream just to get a single image.
```

Comment:
That's exactly how this patch works yes. Instead of spinning up:
* stream = camera.Stream()
* stream.Next()
* stream.Close

We first ask "Hey camera, do you have a "Read" method?" Great -- we'll just call that one time and avoid this stream non-sense.